Automattic/crowdsignal-ui
# Update project renderer after block names change

This PR updates the project-renderer workspace to be able to render after some refactors.

## Test instructions
Checkout and run `yarn install && yarn workspace @crowdsignal/project-renderer start`, browse http://crowdsignal.localhost:9000

Albeit ugly, the hardcoded project should render
